hoff.nix

```diff
@@ -0,0 +1,86 @@
+{ lib, haskellPackages, nix-gitignore, git, coreutils, openssh }:
+  haskellPackages.mkDerivation {
+    pname = "hoff";
+    version = "0.25";
+
+    src =
+      let
+        # We do not want to include all files, because that leads to a lot of things that nix
+        # has to copy to the temporary build directory that we don't want to have in there
+        # (e.g. the `.stack-work` directory, the `.git` directory, etc.)
+        prefixWhitelist = builtins.map builtins.toString [
+          ./app
+          ./package
```

Comment:
`./package` contains scripts related to building the .deb, is it still needed for the Nix build?

Reply:
Yes, the folder contains a file that is used in the tests (`package/example-config.json`)